umbraco/Umbraco.CMS.Backoffice
# Proff of concept for navigating to edit path when created

Lazy load workspace editor.
Navigate to the edit URL when created.

**Test steps:**
Create a new language
See the URL changed to edit/:isoCode when creating was succesful.
